src/components/Calculator.js

```diff
@@ -1,83 +1,101 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Calculator.css';
+import calculate from '../logic/calculate';
 
 export default function Calculator() {
+  const [theState, setTheState] = useState({
+    total: 0,
+    next: 0,
+    operation: null,
+  });
+
+  const handleEvent = (e) => {
+    const result = calculate(theState, e.target.value);
+    setTheState(result);
+  };
+
+  const handleChange = (e) => {
+    setTheState({
+      total: e.target.value,
+    });
+  };
+
   return (
     <div className="text-[#333] text-sm font-bold p-12">
       <table className="h-[300px] w-[350px] mx-auto bg-[#E0E0E0]">
         <thead>
           <tr>
             <td colSpan="4">
-              <input type="text" value="0" className="w-full pr-1 text-right text-xl font-normal text-white bg-[#858693]" />
+              <input type="text" onChange={handleChange} value={theState.total || theState.next || 0} className="w-full pr-1 text-right text-xl font-normal text-white bg-[#858693]" />
```

Comment:
- [ ] @yusufobr great work on your app, loved the simplicity of your implementation, however, I noted something small, when am adding two numbers eg 1 and 2 when I click 2 after the equal sign it's not displayed, this is due to the pattern of displayed values kindly change ` {theState.total || theState.next || 0}` to `{theState.next || theState.total || 0}` to fix the issue